speechbrain/pretrained/interfaces_topk.py

```diff
@@ -0,0 +1,533 @@
+"""Example extension of interfaces; it's a show-casing Easter Egg.
```

Comment:
This has to be put in another PR and we have to check if  we want it or not. Especially since it's impacting most recipes. It should be a edge case, not a standard. Or if it's standard, we should make the use of it in other recipe look better.

Reply:
I dropped the file. Will open a PR after the big release.